### DISCUSSION TITLE
Custom backgrounds for inline code?

### DISCUSSION THREAD

**Discussion by tracyhenry** [gonzalezreal/swift-markdown-ui#278] [General]
Hi, amazing library, thanks for building this!

I wanted to replicate how Github renders inline code like the following:

---
<img width="245" alt="Screenshot 2023-11-28 at 11 16 01 PM" src="https://github.com/gonzalezreal/swift-markdown-ui/assets/5410441/8b360c81-fcff-4991-ab44-28f233032d9f">

---

But there doesn't seem to be way to do it. There's no block style configuration for `.code`. The closest I could get was using the `BackgroundColor` text style, which doesn't have any padding or rounded corners for the background:
<img width="455" alt="Screenshot 2023-11-28 at 11 09 50 PM" src="https://github.com/gonzalezreal/swift-markdown-ui/assets/5410441/13d10685-fc8c-4fdf-9df8-31c974639c06">

Wonder if this will be supported?

**Comment by gonzalezreal** [comment]
Hi @tracyhenry,

It is not possible at the moment. I am working on something to make this and other features possible in the future. Stay tuned.
